torch/nn/modules/loss.py

```diff
@@ -484,20 +500,17 @@ class BCEWithLogitsLoss(_Loss):
         >>> output = loss(input, target)
         >>> output.backward()
     """
-    def __init__(self, weight=None, size_average=True, reduce=True):
+    def __init__(self, weight=None, size_average=True, reduce=True, pos_weight=None):
         super(BCEWithLogitsLoss, self).__init__(size_average, reduce)
         self.register_buffer('weight', weight)
+        self.register_buffer('pos_weight', pos_weight)
```

Comment:
This will break backward compatibility. We need to implement `__setattr__`, and some extra care around state dicts might be necessary too.

Reply:
Is it needed for loading state dict? I'm not sure if I properly understand the problem.

Reply:
@apaszke I don't think that this will break BC because we don't save & load None buffers anyways.